test/jdk/java/nio/channels/DatagramChannel/AdaptorMulticasting.java

```diff
@@ -419,7 +407,13 @@ static void testSendReceive(MulticastSocket s, InetAddress group) throws IOExcep
         // send message to multicast group
         DatagramPacket p = new DatagramPacket(message, message.length);
         p.setSocketAddress(target);
-        s.send(p, (byte) 1);
+        final int oldTTL = s.getTimeToLive();
```

Comment:
Just replacing it with s.send(p) should be okay here, no need to set/restore TTL.